products/macos.md

```diff
@@ -25,22 +25,22 @@ releases:
     eol: false
     link: https://support.apple.com/HT213268
     releaseDate: 2022-10-24
-    latestReleaseDate: 2022-12-13
-    latest: '13.1'
+    latestReleaseDate: 2023-01-23
```

Comment:
Latest link looks not correct, should be https://support.apple.com/en-euro/HT213605 ?

Reply:
After having a look at the page I think it is just not updated for the time being. Closing this issue.